ant-global.xml

```diff
@@ -586,20 +619,4 @@
     <target name="dist.dependencies">
          <antcall target="depend.${ant.project.name}"/>
     </target>
```

Comment:
Thanks
created separate commit for this change, not adding any jira ticket number as this change is not reported anywhere